test/cppsuite/create_test.sh

```diff
@@ -8,10 +8,10 @@ if [ $# -eq 0 ]
 fi
 
 # Check the test name
-if [[ $1 =~ ^[0-9a-zA-Z_-]+$ ]];then
+if [[ $1 =~ ^[a-zA-Z]+(_[0-9a-zA-Z]+)$ ]]; then
```

Comment:
Looks good! I'd recommend also removing the upper case letters so we align to the `snake_case` naming of our classes: `^[a-z][_a-z0-9]+$`

Reply:
thanks Andrew - Will change that now